engine/src/main/java/org/terasology/rendering/opengl/PBO.java

```diff
@@ -30,7 +30,8 @@
  * Created by manu on 15.03.2015.
  */
 public class PBO {
-    private int pboId;
+    private ByteBuffer resultBuffer = null;
```

Comment:
As far as I know PBOs are always 1x1 pixels and are created only once. I appreciate the optimization effort but I feel this particular portion of code is getting more complicated for negligible gains. Unless we start to use PBOs more intensively, I'd recommend not to go ahead with these particular changes.